### ISSUE TITLE
Animated.modulo does not call event listener

### ISSUE THREAD

**Issue by macrozone** [necolas/react-native-web#2513]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

Animated.modulo does not call event listeners on update.

This codesandbox shows the issue: https://codesandbox.io/s/vibrant-lumiere-6p71s1?file=/src/App.js

the same code does run the listener for both AnimatedValues on react-native, but only the first one on react-native-web

### Expected behavior

Animated.modulo should call addEventListener

### Steps to reproduce

see https://codesandbox.io/s/vibrant-lumiere-6p71s1?file=/src/App.js

### Test case

https://codesandbox.io/s/vibrant-lumiere-6p71s1?file=/src/App.js

**Comment by necolas** [comment]
Yeah it's really annoying to sync modules like Animated, which regularly has bugs and needs patching in RN. See this issue for being able to take on the same version of Animated as is found in RN https://github.com/facebook/react-native/issues/35263

**Comment by macrozone** [comment]
> Yeah it's really annoying to sync modules like Animated, which regularly has bugs and needs patching in RN. See this issue for being able to take on the same version of Animated as is found in RN [facebook/react-native#35263](https://github.com/facebook/react-native/issues/35263)

I see, but I think the Bug may be fixed without the refactorings in react-native first.

I tried to debug it in my project, but did not find yet the culprit.

is there a way to test and try to reproduce it in react-native-web repo directly?  Then i would first try to fix it.

**Comment by macrozone** [comment]
i checked again, actually the error is already present in react-native itself

**Comment by macrozone** [comment]
created an upstream issue https://github.com/facebook/react-native/issues/37061

**Comment by necolas** [comment]
Thanks!
